src/main/groovy/org/arquillian/spacelift/gradle/SpaceliftPlugin.groovy

```diff
@@ -142,6 +148,7 @@ class SpaceliftPlugin implements Plugin<Project> {
             task << {
                 println "Spacelift Workspace:          ${project.spacelift.workspace}"
                 println "Spacelift Installations Cache Dir:  ${project.spacelift.installationsDir}"
+                println "Spacelift configuration: \n${project.spacelift.configuration.toString()}"
```

Comment:
No, only the main configuration block. Each profile block then has its own configuration block. The description is not aware of selected profile afaik.